perllib/FixMyStreet/App/Controller/My.pm

```diff
@@ -382,10 +382,12 @@ sub notify_preference : Local : Args(0) {
 
     my $update_notify = $c->get_param('update_notify');
     my $alert_notify = $c->get_param('alert_notify');
+    my $receive_questionnaires = $c->get_param('receive_questionnaires');
```

Comment:
This works fine, but wondered if `questionnaire_notify` would be keeping in sync with the naming of the other similar variables `update_notify` and `alert_notify`?